sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/ServiceBusModelFactory.cs

```diff
@@ -17,30 +19,78 @@ public static class ServiceBusModelFactory
         /// </summary>
         [EditorBrowsable(EditorBrowsableState.Never)]
         public static ServiceBusReceivedMessage ServiceBusReceivedMessage(
-            ServiceBusMessage sentMessage,
-            bool isSettled,
-            int deliveryCount,
-            DateTimeOffset lockedUntil,
-            long sequenceNumber,
-            string deadLetterSource,
-            short partitionId,
-            long enqueuedSequenceNumber,
-            DateTimeOffset enqueuedTime,
-            Guid lockTokenGuid,
-            object bodyObject
-        ) => new ServiceBusReceivedMessage
+            BinaryData body = default,
+            string messageId = default,
+            string partitionKey = default,
+            string viaPartitionKey = default,
+            string sessionId = default,
+            string replyToSessionId = default,
+            TimeSpan timeToLive = default,
+            string correlationId = default,
+            string label = default,
+            string to = default,
+            string contentType = default,
+            string replyTo = default,
+            DateTimeOffset scheduledEnqueueTime = default,
+            IDictionary<string, object> properties = default,
+            Guid lockTokenGuid = default,
+            int deliveryCount = default,
+            DateTimeOffset lockedUntil = default,
+            long sequenceNumber = -1,
```

Comment:
Hmm, why -1 as opposed to default?

Reply:
-1 is the default on the ReceivedMessage itself.